frontend/src/components/auth/PrivateRoute.tsx

```diff
@@ -28,23 +28,23 @@ const PrivateRoute: React.FC<PrivateRouteProps> = ({
     return authenticatedUser?.role === Role.ADMIN ? (
       <Route path={path} exact={exact} component={component} />
     ) : (
-      <Redirect to={Routes.LANDING_PAGE} />
+      <Redirect to={Routes.LOGIN_PAGE} />
```

Comment:
not sure if we want to make this change everywhere... also for certain volunteer pages, should they only be accessible when they are approved? e.g. `VOLUNTEER_SHIFTS_PAGE`